projects/Mallard/Makefile

```diff
@@ -11,4 +11,10 @@ debug-ios:
 
 beta-ios:
 	yarn bundle-ios
-	cd ios && fastlane beta
+	fastlane ios beta
+
+beta-android:
+	mkdir -p android/app/src/main/assets
+	yarn bundle-android
+	rm -rf android/app/src/main/res/drawable-*
```

Comment:
Hackily remove these directories to avoid a [duplicate resource error](https://stackoverflow.com/questions/53239705/react-native-error-duplicate-resources-android). This will be [fixed properly](https://github.com/facebook/react-native/commit/962437fafd02c936754d1e992479056577cafd05) in `react-native v0.60`